packages/dotcom-ui-flags/src/server/formatFlagsJSON.ts

```diff
@@ -1,6 +1,6 @@
 import { TFlagsData } from '../types'
 
-export default function formatFlagsJSON(flags: TFlagsData = {}): string {
+export default function formatFlagsJSON(flags: TFlagsData = {}): object {
```

Comment:
The returned value will also be of type `TFlagsData` now, right? So `function formatFlagsJSON(flags: TFlagsData = {}): TFlagsData` will work, I think.